Gemfile

```diff
@@ -12,3 +12,5 @@ gem "sprockets-rails"
 # gem "debug", ">= 1.0.0"
 
 gem "devise"
+gem "omniauth"
+gem "omniauth-github"
```

Comment:
@tiagocasanovapt No need to add this here. `gemspec` already defines devise and omniauth and omniauth-github as dependencies and you are including it in line 5.